tests/generation/test_utils.py

```diff
@@ -1073,6 +1073,9 @@ def test_beam_search_generate_dict_outputs_use_cache(self):
     @require_torch_multi_accelerator
     def test_model_parallel_beam_search(self):
         for model_class in self.all_generative_model_classes:
+            if torch_device == "xpu":
```

Comment:
is better, as the device can be numbered

```suggestion
            if "xpu" in torch_device:
```